website/src/components/LeaderBoardItem.tsx

```diff
@@ -0,0 +1,12 @@
+const RankItem = ({ username, score }) => {
```

Comment:
Could the class get the same name as the file?  I think it'll be less surprising when importing.